.travis.yml

```diff
@@ -0,0 +1,11 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+sudo: required 
+dist: trusty 
+language: generic 
+env:
+  - ROS_DISTRO="kinetic"
+install:
+  - git clone --depth 1 https://github.com/ros-industrial/industrial_ci.git .industrial_ci  
```

Comment:
you probably need all the packages from here https://bluesat.atlassian.net/wiki/spaces/OWRT/pages/4620426/Required+Ubuntu+Packages 

Plus step 6 of this: https://bluesat.atlassian.net/wiki/spaces/OWRT/pages/3211577/Getting+Started+with+Software

(just going through all the prs and saw this :stuck_out_tongue: , they're not part of the default ros install)